operate/cli.py

```diff
@@ -650,7 +656,7 @@ async def _deploy_service_onchain(request: Request) -> JSONResponse:
         operate.service_manager().deploy_service_onchain(
             hash=request.path_params["service"]
         )
-        operate.service_manager().stake_service_on_chain(
+        operate.service_manager().stake_service_on_chain(  # pylint: disable=no-value-for-parameter
```

Comment:
The definition seems to require more args:

https://github.com/valory-xyz/optimus-quickstart/blob/bec0de58048f7712e30fccaa2bdc0530ad4dc14c/operate/services/manage.py#L888

Reply:
the endpoint not used yet and the [function](https://github.com/valory-xyz/optimus-quickstart/blob/bec0de58048f7712e30fccaa2bdc0530ad4dc14c/operate/services/manage.py#L888)  in not implemented